spec/support/factories.cr

```diff
@@ -16,6 +16,9 @@ end
 def create_git_repository(project, *versions)
   Dir.cd(tmp_path) do
     run "git init #{Process.quote(project)}"
+    Dir.cd(Process.quote(project)) do
+       run "git branch -m master"
```

Comment:
I believe this needs to be `git checkout --orphan master` because at this point there exists no branch that could be moved with `branch -m`.

Reply:
Both worked on my system, but  if you prefer the latter then here it comes.

Reply:
Can we use `main`? 🙏

Reply:
The test originally assumed that the default branch is "master". This is not the case for everyone any more. (e.g. I have a default branch called "main"). This PR fixes an assumption of the test that is not correct any more.

If you'd like to change the test to use some other branch name, that, IMHO should be a separate PR.